assembl/static2/js/app/components/common/ideaPreview.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { Translate } from 'react-redux-i18n';
+import Statistic from './statistic';
+
+class IdeaPreview extends React.Component {
```

Comment:
OK I just converted this component to a functional component (but from my current knowledge I'm really not sure it's worth the time spent)